doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1136,6 +1096,7 @@ Deprecations
 - :func:`pandas.types.is_datetimetz` is deprecated in favor of `pandas.types.is_datetime64tz` (:issue:`23917`)
 - Creating a :class:`TimedeltaIndex` or :class:`DatetimeIndex` by passing range arguments `start`, `end`, and `periods` is deprecated in favor of :func:`timedelta_range` and :func:`date_range` (:issue:`23919`)
 - Passing a string alias like ``'datetime64[ns, UTC]'`` as the `unit` parameter to :class:`DatetimeTZDtype` is deprecated. Use :class:`DatetimeTZDtype.construct_from_string` instead (:issue:`23990`).
+- Operations where the offset alias ``'D'`` or :class:`Day` acts as a fixed, absolute duration of 24 hours are deprecated. This includes :class:`Timestamp` arithmetic and all operations with :class:`Timedelta`. :class:`Day` will always represent a calendar day in a future version (:issue:`22864`)
```

Comment:
Can you give some examples here? 

For example, with the following I don't see a change or warning:

```
In [11]: pd.Timestamp("2012-10-28", tz='Europe/Brussels') + pd.offsets.Day()                                                                                                    
Out[11]: Timestamp('2012-10-28 23:00:00+0100', tz='Europe/Brussels')
```